backend/persistent/datastores/drivers/AWSAthenaDriver.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+import Logger from '../../../logger';
+const NUMBER_OF_RETRIES = 5;
+
+/**
+ * The following function will create an AWS Athena Client
+ * @param {object} connection - AWS Athena Connection Parameters
+ * @param {string} connection.accessKey - AWS Access Key
+ * @param {string} connection.secretAccessKey - AWS Secret Key
+ * @param {string} connection.region - AWS Region
+ * @returns {object} AWS Athena Client
+ */
+export function createAthenaClient(connection) {
+    const connectionParams = {
+        apiVersion: '2017-05-18',
+        accessKeyId: connection.accessKey,
+        secretAccessKey: connection.secretAccessKey,
+        region: connection.region,
+        maxRetries: 5
+    };
+    const athenaClient = new AWS.Athena(connectionParams);
+
+    return athenaClient;
+}
+
+/**
+ * The following method will execute the sql statement to query the
+ * athena database
+ * @param {object} athenaClient  - Object created using create athena client
+ * @param {object} params - Connection Parameters
+ * @param {string} params.dbName - Database name to connect to
+ * @param {string} params.sqlStatement - SQL statement to execute
+ * @param {string} params.s3Outputlocation - Location will Athena will output resutls of query
+ * @return {string} requestId
+ */
+export function startQuery(athenaClient, params) {
+    const client = athenaClient;
+
+    const queryParams = {
+        QueryString: params.sqlStatement,
+        ResultConfiguration: {
+          OutputLocation: params.s3Outputlocation,
```

Comment:
How about `params` uses the same property names AWS does?
Just to keep things simple.

Reply:
The reason I set it to S3 Output Location was to make it easier for users to who using Falcon.  If your previous comments there was confusion about what Output Location was.  I named the parameter to be S3 Output Location hoping that it would make it easier for Falcon users.  If it is causing confusion then I can update the parameter names

Reply:
Here, I had developers in mind. If we use the same names, then developers won't have to learn that `s3Outputlocation` corresponds to `OutputLocation`.

This is a minor point (not a blocking request).